docs-gen/content/rule_set/data_entry/data_types.md

```diff
@@ -23,3 +23,38 @@ String     | character string           | n/a  | n/a
 ByteBuffer | buffer of bytes (aka BLOB) | n/a | n/a
 
 Datatypes are not specified for ```branch```
+
+
+
+## Data Streams
+
+Data Entries, which describe sensors offering binary streams
+(e.g. cameras), are not supported directly by VSS with a
+dedicated data type. Instead, they are described through the
+meta data about the sensor itself and how to retrieve the
+corresponding data stream.
+
+A camera can be a good example to it. The Data Entry for the camera
+and the corresponding video stream could look like:
+
+```YAML
+- Camera:
+  type: branch
+  description: Information about the camera and how to connect to the video stream
+
+- Camera.IsActive:
+  type: actuator
+  datatype: boolean
+  description: If the camera is active, the client is able to retrieve the video stream
+
+- Camera.URI:
+  type: sensor
+  datatype: string
+  description: URI for retrieving the video stream
```

Comment:
The description should give information about data format / streaming protocol.

Reply:
This should be in a different node then. My thinking was mostly, that the protocol is part of the URI. 

But we could make it explicit with a node, like
```YAML
- camera.Protocol
   type: attribute
   datatype: string
   description: Protocol used for the video stream, connection done via Camera.URI
```

In private discussions that came up already once before, so I'll add it to the PR.

Reply:
> This should be in a different node then

Why?  My proposal was only that the description should explain everything you need to know to use the data item (in this case the URL).   It is generally the case that the description needs to give enough information.  In this case you have to explain what is actually being transferred.

I don't think Protocol needs to be a data item in VSS.  It would only need to be a separate item if it is something that can change dynamically, and that is usually not the case.

If "camera" was part of the standard VSS catalog, we might consider being a bit more formal.  But this is just a guideline for how anyone can define streaming URLs, for any purpose.  In my opinion there is no reason to have a _guideline_ to put Protocol in a separate attribute for every definition of a streaming URL.

Reply:
> My thinking was mostly, that the protocol is part of the URI.

On some protocol level it is, of course.  But in reality behind **http://something** lies a lot of different data formats typically, even sometimes some embedded "protocols". 
 In particular, "the video stream" that was mentioned in the comment is not enough to define it clearly.   It might not always make sense to describe these things, but in some cases it should and the video stream was a particular case, in my opinion.

Reply:
@danielwilms: add description, remove protocol node for simplicity